### PR TITLE
New package: Nbc66.VTFEditReloaded version 2.0.4.00

### PR DESCRIPTION
**Pull request by Nbc66**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/139610)

### CODE REVIEW

**Comment by Nbc66** [comment]
@microsoft-github-policy-service agree

**Comment by Nbc66** [comment]
Hi is there anything else i need to do to be abel to get this package inside of winget?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 791.)

**Comment by Nbc66** [comment]
> Automatic Validation ended with:
> 
> > No errors to post.
> 
> (Automated response - build 791.)

So whats the issue then?

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/n/Nbc66/VTFEditReloaded/2.0.4.00/Nbc66.VTFEditReloaded.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Nbc66.VTFEditReloaded
+PackageVersion: 2.0.4.00
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/n/Nbc66/VTFEditReloaded/2.0.4.00/Nbc66.VTFEditReloaded.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2010.x64
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
With both of these, it launches normally:
image

**Comment by Nbc66** [comment]
@stephengillie I updated the manifest with the requested changes

**Review by stephengillie — APPROVED** [review]

**Pull request merged**